games/pacman.puml

```diff
@@ -0,0 +1,186 @@
+@startuml
+package "Process Input Handler" {
+!includesub ../uml/frontend-software-system/process-input-handler.puml!INTERFACE
+}
+package "Render Handler" {
+!includesub ../uml/frontend-software-system/render-handler.puml!INTERFACE
+}
+!include ../uml/frontend-software-system/game-settings.puml
+
+
+package Pac-Man {
+
+class Position {
+ - x : int
+ - y : int
+}
+
+class Wall {
+ - sprite : Sprite
+ - position: Position
+}
+
+class PowerSphere
+{
+ - sprite : Sprite
+ - time : int
+ - position: Position
+ + collect() : int
+ + activatePower() : void
+}
+
+class Sphere
+{
+ - sprite : Sprite
+ - position: Position
+ + collect() : int
+}
+
+abstract class Ghost
+{
+ - sprite : Sprite
+ -state : string
+ -color : string
```

Comment:
What's the purpose of the color attribute?

Reply:
was an error.